client/components/editor/teaching-elements/Audio.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <div class="te-audio">
+    <div v-show="showPlaceholder">
+      <div class="audio-placeholder">
+        <div class="message">
+          <span class="heading">Audio placeholder</span>
+          <p v-if="!isFocused">Select to edit</p>
+          <p v-else>Please use toolbar to enter url</p>
+        </div>
+      </div>
+    </div>
+    <div v-show="!showPlaceholder">
+      <div v-if="!isFocused && !hasError" class="overlay">
+        <div class="message">Click to preview</div>
+      </div>
+      <div class="audio-container">
+        <audio
+          v-show="!hasError"
+          :src="source"
+          ref="audio"
+          controls>
+          This browser does not support HTML5 audio element.
+        </audio>
+        <div v-if="hasError" class="error">
+          <div class="message">
+            <span class="icon mdi mdi-alert"></span>
+            <p>Error loading audio file!</p>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'te-audio',
+  props: {
+    element: { type: Object, required: true },
+    isFocused: { type: Boolean, default: false }
+  },
+  data() {
+    return {
+      hasError: false
+    };
+  },
+  computed: {
+    audioElement() {
+      return this.$refs.audio;
+    },
+    source() {
+      return this.element.data.url;
+    },
+    showPlaceholder() {
+      return !this.source;
+    }
+  },
+  methods: {
+    playAudio() {
+      this.audioElement.play();
+    },
+    pauseAudio() {
+      this.audioElement.pause();
+    },
+    checkError() {
+      const { error } = this.audioElement;
+      this.hasError = !!error;
+    }
+  },
+  watch: {
+    isFocused() {
+      if (this.isFocused) {
+        return this.playAudio();
+      }
+      return this.pauseAudio();
```

Comment:
_Play on focus_ is one possible approach but it is not really aligned with existing video component. Please add some UI layer to enable manual playback control.
Pick your poison: https://github.com/vuejs/awesome-vue#audio--video 😉

Reply:
Are the standard HTML audio controls enough (play/pause, seek, volume)? Because that is what gets rendered to the user. This just triggers play when you focus the element.

Reply:
If they look same across different browsers 🤔